README.md

```diff
@@ -1,49 +1,30 @@
-## contexture-react
-`contexture-react` is a suite of react components for building contexture interfaces
+# contexture-react
 
+React components for building contexture interfaces https://smartprocure.github.io/contexture-react
 
 [![CircleCI](https://circleci.com/gh/smartprocure/contexture-react.svg?style=svg)](https://circleci.com/gh/smartprocure/contexture-react)
 [![Greenkeeper badge](https://badges.greenkeeper.io/smartprocure/contexture-react.svg)](https://greenkeeper.io/)
 [![npm version](https://badge.fury.io/js/contexture-react.svg)](https://badge.fury.io/js/contexture-react)
 ![dependencies](https://david-dm.org/smartprocure/contexture-react.svg)
 [![Storybook](https://img.shields.io/badge/BUILT_WITH-STORYBOOK-f1618c.svg?style=for-the-badge)](https://smartprocure.github.io/contexture-react)
 
-
 # Documentation
 
-Documentation is largely provided via our [Storybook](https://smartprocure.github.io/contexture-react).
+Documentation lives under `./docs` and our [Storybook](https://smartprocure.github.io/contexture-react)**.
 
 **The most complete example is the [Live Demo using IMDB data and the Grey Vest theme](https://smartprocure.github.io/contexture-react/?path=/story/live-demos-imdb--grey-vest-theme)**
 
-
-
-## Theming
```

Comment:
it might be a good idea to keep this section just for a link to the theming doc in storybook (or directly in the source, since storybook's default markdown styling sucks). Not a big deal, but I think having a section for it in the readme helps communicate up-front that we have theming support, without the user having to go look at the docs/stories to find out.

Reply:
I like linking to the storybook's theming docs. It'll get better once we start using storydocs